content/en/news/releases/1.22.x/announcing-1.22/_index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing Istio 1.22.0
+linktitle: 1.22.0
+subtitle: Major Release
+description: Istio 1.22 Release Announcement.
+publishdate: 2024-05-13
+release: 1.22.0
+aliases:
+- /news/announcing-1.22
+- /news/announcing-1.22.0
+---
+
+We are pleased to announce the release of Istio 1.22. This is the first Istio release of 2024. We would like to thank the
```

Comment:
This was already said in 1.21. ^^

https://istio.io/latest/news/releases/1.21.x/announcing-1.21/

Reply:
copy from that page, removed.